### PR TITLE
Publish docs and apps on website

### PR DESCRIPTION
**Pull request by cboulanger**
This publishes the new docs and the new demo apps in the `6.0.0-beta/docs` and `6.0.0-beta/apps` subfolders of the homepage, respectively.

### CODE REVIEW

**Review by derrell — APPROVED** [review]

File: .travis/build-site-v6 [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+TARGET=deploy/$(cat ./version.txt)
+mkdir -p $TARGET
+
+[ -d ./node_modules/@qooxdoo/compiler ] || npm i --no-save @qooxdoo/compiler
+
+# build apps (unless already built) & copy to website target
+[ -d ./apps/resource ] || exe npx qx compile compile.json --target build  --feedback=0
+exe cp -a ./apps $TARGET
+exe find $TARGET -type f -name '*.map' -delete
```
Comment (hkollmann) [inline .travis/build-site-v6]: Should'nt we put this into a compile.js file so that it works for npm_deploy too?

**Review by hkollmann — COMMENTED** [review]
Do we have a doc page with a list off all apps? Otherwise they are not reachable yet...

**Review by hkollmann — APPROVED** [review]

File: .travis/build-site-v6 (hunk 1 above)
Comment (cboulanger) [reply .travis/build-site-v6]: Yes, this could be DRYed up with `$TARGET` being a CLI parameter - but let's test it "live" first, then optimize later.

**Review by johnspackman — APPROVED** [review]

**Comment by cboulanger** [comment]
@hkollmann Not yet, but I will link to the demo apps from everywhere in the documentation once the PR is merged, and we can also add a special page in the docs.

File: .travis/build-site-v6 (hunk 1 above)
Comment (hkollmann) [reply .travis/build-site-v6]: aggreed

**Review by oetiker — APPROVED** [review]

**Review by zaucker — APPROVED** [review]

**Pull request merged**